src/encoded/static/components/app.js

```diff
@@ -384,13 +384,21 @@ export default class App extends React.PureComponent {
 
```

Comment:
Looks like there are a lot of random formatting changes in this file. I'll try to come into the office at some point soon so we can compare our prettier & vscode configurations.